src/components/subscriptions/SubscriptionZeroStateMessaging.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import { Card } from '@edx/paragon';
+import InviteLearnersButton from './buttons/InviteLearnersButton';
+import { TAB_PENDING_USERS } from './data/constants';
+import { SubscriptionContext } from './SubscriptionData';
+import { ToastsContext } from '../Toasts';
+import { SubscriptionDetailContext } from './SubscriptionDetailContextProvider';
+
+const SubscriptionZeroStateMessaging = () => {
```

Comment:
just a pref thing I suppose, but perhaps this name can be more component like such as SubscriptionZeroStateCTA or SubscriptionZeroStateMessage ? not too strong a feeling though.